doc/source/whatsnew/v0.17.0.txt

```diff
@@ -136,21 +136,22 @@ numeric types when there were no values convertible to a numeric type.  For exam
      0  a
      1  b
 
-returns the original DataFrame with no conversion. This change alters
-this behavior so that
+  returns the original DataFrame with no conversion. This change alters
+  this behavior so that
 
   .. ipython:: python
 
     pd.DataFrame({'s': ['a','b']})
     df.convert_objects(numeric=True, coerce=True)
 
-converts all non-number-like strings to ``NaN``.
+    converts all non-number-like strings to ``NaN``.
 
 - In earlier versions of pandas, the default behavior was to try and convert
-datetimes and timestamps. The new default is for ``DataFrame.convert_objects``
-to do nothing, and so it is necessary to pass at least one conversion target
-in the method call.
+  datetimes and timestamps. The new default is for ``DataFrame.convert_objects``
+  to do nothing, and so it is necessary to pass at least one conversion target
+  in the method call.
 
+.. _whatsnew_0170.api_breaking:
```

Comment:
can you make this api_breaking.other ? (you switched the api_breaking and api_breaking.other)